api/lib/database/methods.js

```diff
@@ -248,6 +250,27 @@ const getNetwork = ({ hasura_url, hasura_admin_key }) => () => async (id) => {
   }
 }
 
+const getNetworkLinks = ({ hasura_url, hasura_admin_key }) => () => async (
+  id
+) => {
+  const {
+    data: { networks }
+  } = await query({
+    hasura_url,
+    hasura_admin_key
+  })(`
+    query {
+      networks: networks(where: { 
+        id: {_eq: "${id}"}
+      }) {
+        links
```

Comment:
should we put this in another table: `networkGovernance` so we can normalize the table?

networkGovernance
- networkId
- link
- linkCaption

Reply:
I think it is more organized per network? Otherwise it would look messy, don't you think, with multiple repetitions of networkId per each network link... what would the primary key be?
I think it is better to have all links in an array within the network. But we could also do `networkId` and `links` (array) in this new table, `networkGovernance`

Reply:
What is your opinion on the arrays being hard to read and edit in the db?

Reply:
IMO, it is worth to get the hang of it. Somethings it is just cleaner & better to store them like that, like for example the cryptos and the links in the Network Object. Plus you already have those as examples there

Reply:
Why is it cleaner and "better" to store them this way?

Reply:
because you get all you need in one place without needing multiple calls (more code to maintain + slower). You get your beautiful object,
```
[
{
name: "Edwards curve",
curve: "ed25519"
},
{
name: "Schnorrkel curve",
curve: "sr25519"
},
}
```

Otherwise you would need one column called curveName0 and another one called curveCurve0, and so on and so forth. Truly horrible, if you want my opinion

Reply:
no, you can create a table `networkCrypto` then this table has columns:
- networkId
- curve
- name
- default

this way you can easily edit all of the rows.